src/test/java/io/specto/hoverfly/ruletest/HoverflyRuleRemoteInstanceTest.java

```diff
@@ -3,18 +3,51 @@
 import io.specto.hoverfly.junit.rule.HoverflyRule;
 import org.junit.ClassRule;
 import org.junit.Ignore;
+import org.junit.Test;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
 
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import static io.specto.hoverfly.junit.core.HoverflyConfig.authenticationConfigs;
 import static io.specto.hoverfly.junit.core.HoverflyConfig.configs;
 import static io.specto.hoverfly.junit.core.SimulationSource.classpath;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpStatus.CREATED;
+import static org.springframework.http.MediaType.APPLICATION_JSON;
 
-
-@Ignore("For on demand end-to-end testing")
-public class HoverflyRuleRemoteInstanceTest extends BaseHoverflyRuleTest {
+@Ignore
+public class HoverflyRuleRemoteInstanceTest {
```

Comment:
I think it would be better to actually start a Hoverfly locally, but config it as a remote in the rule